Content.Server/GameTicking/Rules/ThiefRuleSystem.cs

```diff
@@ -92,7 +89,7 @@ private void DoThiefStart(ThiefRuleComponent component)
         }
     }
 
-    public bool MakeThief(ICommonSession thief, bool AddPacified)
+    public bool MakeThief(ICommonSession thief, bool addPacified)
     {
         var thiefRule = EntityQuery<ThiefRuleComponent>().FirstOrDefault();
         if (thiefRule == null)
```

Comment:
MakeThief still needs to tie the thief to the specified rule and not a random one.